src/transformers/models/omdet_turbo/configuration_omdet_turbo.py

```diff
@@ -0,0 +1,311 @@
+            The activation dropout rate of the encoder ffn.
+        encoder_in_channels (`list(int)`, *optional*, defaults to `[192, 384, 768]`):
+            The input channels for the encoder.
+        encoder_feat_strides (`list(int)`, *optional*, defaults to `[8, 16, 32]`):
+            The feature strides for the encoder.
+        encode_proj_layers (`list(int)`, *optional*, defaults to `[2]`):
+            The projection layers for the encoder.
+        encoder_attention_heads (`int`, *optional*, defaults to 8):
+            The number of attention heads for the encoder.
+        normalize_before (`bool`, *optional*, defaults to `False`):
+            Whether to normalize before in the encoder.
+        eval_size (`Tuple[int, int]`, *optional*):
+            Height and width used to computes the effective height and width of the position embeddings after taking
+            into account the stride (see RTDetr).
+        encoder_layers (`int`, *optional*, defaults to 1):
+            The number of layers in the encoder.
+        positional_encoding_temperature (`int`, *optional*, defaults to 10000):
+            The positional encoding temperature in the encoder.
+        encoder_dim_feedforward (`int`, *optional*, defaults to 2048):
+            The feedforward dimension for the encoder.
+        decoder_num_heads (`int`, *optional*, defaults to 8):
+            The number of heads for the decoder.
+        decoder_num_layers (`int`, *optional*, defaults to 6):
+            The number of layers for the decoder.
+        label_dim (`int`, *optional*, defaults to 512):
+            The dimension of the labels embeddings.
+        class_distance_type (`str`, *optional*, defaults to `"cosine"`):
+            The type of of distance to compare predicted classes to projected labels embeddings.
+        decoder_activation (`str`, *optional*, defaults to `"relu"`):
+            The activation function for the decoder.
+        decoder_encoder_dim_feedforward (`int`, *optional*, defaults to 1024):
+            The feedforward dimension for the task encoder in the decoder.
+        decoder_dim_feedforward (`int`, *optional*, defaults to 2048):
+            The feedforward dimension for the decoder.
+        decoder_num_points (`int`, *optional*, defaults to 4):
+            The number of points sampled in the decoder multi-scale deformable attention module.
+        decoder_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout rate for the decoder.
+        decoder_eval_idx (`int`, *optional*, defaults to -1):
+            Debugging tool to stop the decoder at a certain layer index during evaluation.
+        learn_init_query (`bool`, *optional*, defaults to `False`):
+            Whether to learn the initial query.
+        fuse_type (`str`, *optional*, defaults to `"merged_attn"`):
+            The type of fusion between task and vision embeddings in the decoder.
```

Comment:
Not sure to be honest, no other options are provided in the original code. I should maybe change this to a flag `use_merged_attention`? Or remove the option altogether? It doesn't seem like it would make sense not to use this

Reply:
If there's not different options available from the original repo we can just remove. The config should have a minimal set of params